repo/packages/K/kubernetes/100/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "upgradesFrom": [
+    "1.0.1-1.9.4-rc1"
+  ],
+  "name": "kubernetes",
+  "version": "v1.0.1-1.9.4",
```

Comment:
I don't remember seeing our versions prefixed with a `v`, so wanted to double-check this is intentional and that we won't have any problems because of this.

Reply:
@MatApple @bmcstdio our version don't have that `v`prefix here.